tests/test_randvars/test_normal.py

```diff
@@ -37,14 +41,14 @@ def setUp(self):
             (
                 np.random.uniform(size=(2, 2)),
                 linops.SymmetricKronecker(
-                    A=np.array([[1.0, 2.0], [2.0, 1.0]]),
+                    A=np.array([[1.0, 2.0], [2.0, 10.0]]),
```

Comment:
this SymmetricKronecker matrix was not positive semi-definite. Now it is. Thus, there are no more warnings in the sampling code.